plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -19,6 +19,10 @@ const blocks = {
 	'add-to-cart-form': {
 		customDir: 'product-elements/add-to-cart-form',
 	},
+	'add-to-cart-with-options': {
+		customDir: 'add-to-cart-with-options',
```

Comment:
If the directory path matches the block slug, I think `customDir` is not needed.

Reply:
Done in [fa2ac3c](https://github.com/woocommerce/woocommerce/pull/53215/commits/fa2ac3c43e5bb12e2ca036651f3b6b365e7cbcd3)